pkg/controller/metrics/metrics.go

```diff
@@ -655,4 +744,44 @@ func (ca *clusterAccumulator) addConditionToMap(cond hivev1.ClusterDeploymentCon
 	ca.conditions[cond][clusterType]++
 }
 
+// readMetricsConfig reads the metrics config from the file pointed to by the MetricsConfigFileEnvVar
+// environment variable.
+func readMetricsConfig() (*metricsconfig.MetricsConfig, error) {
+	path := os.Getenv(constants.MetricsConfigFileEnvVar)
+	config := &metricsconfig.MetricsConfig{}
+	if len(path) == 0 {
+		return config, nil
+	}
```

Comment:
Per [slack](https://coreos.slack.com/archives/CE3ETN3J8/p1651774265655569?thread_ts=1651606872.054189&cid=CE3ETN3J8) please make this an error. It means the hive-operator side did something wrong by not setting the env var for the hive-controllers pod.